rest/.jvm/src/test/scala/io/udash/rest/RestServletTest.scala

```diff
@@ -0,0 +1,352 @@
+package io.udash
+package rest
+
+import com.avsystem.commons.*
+import monix.execution.Scheduler
+import monix.reactive.Observable
+import org.mockito.ArgumentMatchers.argThat
+import org.mockito.Mockito.*
+import org.scalatest.BeforeAndAfterEach
+import org.scalatest.concurrent.{Eventually, ScalaFutures}
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.time.{Millis, Seconds, Span}
+
+import java.io.*
+import javax.servlet.AsyncContext
+import javax.servlet.http.{HttpServletRequest, HttpServletResponse}
+import scala.concurrent.Future
+
+class RestServletTest extends AnyFunSuite with ScalaFutures with Matchers with BeforeAndAfterEach with Eventually {
```

Comment:
Any reason to do all this mocking when most (all?) of this could be tested end to end, as in other REST tests in udash-rest?